vj4/ui/components/ide/ide.styl

```diff
@@ -0,0 +1,279 @@
+@import '../../common/common.inc.styl'
+
+$toolbar-secondary-height = rem(2.1875)
+
+.ide
+  display: table-row
+  width: 100%
+  height: 100%
+
+  .ide__left, .ide__right
```

Comment:
should be placed outside `.ide` because it is already prefixed with the parent class name.